packages/backend/src/nest/tor/tor.service.ts

```diff
@@ -105,11 +105,14 @@ export class Tor extends EventEmitter implements OnModuleInit {
 
           this.interval = setInterval(async () => {
             const log = await this.torControl.sendCommand('GETINFO status/bootstrap-phase')
+            this.logger(log.messages[0])
             if (
               log.messages[0] === '250-status/bootstrap-phase=NOTICE BOOTSTRAP PROGRESS=100 TAG=done SUMMARY="Done"'
             ) {
+              this.logger(`Sending ${SocketActionTypes.TOR_INITIALIZED}`)
               this.serverIoProvider.io.emit(SocketActionTypes.TOR_INITIALIZED)
-
+              this.logger('Attempting to redial peers (if possible)')
+              this.emit(SocketActionTypes.REDIAL_PEERS)
```

Comment:
Are we dialing peers when Tor starts and then re-dialing them when Tor is fully initialized? I wonder if we want to skip the initial dial when libp2p is initialized.

Reply:
This is gonna require more refactor due to the timing/interactions of tor and libp2p.

Reply:
We dial peers when libp2p is ready, but that isn't necessarily when tor is.